### PR TITLE
Homework 3.0

### PR DESCRIPTION
**Pull request by Ievgenii-Balabanov**
Final version

### CODE REVIEW

File: school/diary/views.py [hunk 1]
```diff
@@ -47,42 +47,23 @@ def index(request):
...
-#         position = request.POST.get("position")
-#         club = request.POST.get("contact")
-#         transfer_fee = request.POST.get("transfer_fee")
-#         response = f"Player Name: {name}, Position: {position}, Club: {club}, " \
-#                    f"Transfer fee: {transfer_fee} millions euros"
-#     else:
-#         response = "Please create a player"
-#     return HttpResponse(response)
-
 def login(request):
```
Comment (idegtiarov) [inline school/diary/views.py:50]: Great! What about endpoints for changing the params we had previously in Flask app?

**Review by idegtiarov — CHANGES_REQUESTED** [review]

File: school/diary/views.py (hunk 1 above)
Comment (Ievgenii-Balabanov) [reply school/diary/views.py:50]: You mean endpoints that gives you an opportunity to change some existing parameters in DB?

Comment (idegtiarov) [reply school/diary/views.py:50]: correct

Comment (Ievgenii-Balabanov) [reply school/diary/views.py:50]: if i knew how(

Comment (Ievgenii-Balabanov) [reply school/diary/views.py:50]: parameters can be changed via form?

**Pull request closed without merging**